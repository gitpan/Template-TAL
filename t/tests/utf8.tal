<?xml version="1.0" encoding="utf-8"?>
<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body><span tal:replace="utf8" /></body>
</html>
