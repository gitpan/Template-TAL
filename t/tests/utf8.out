<?xml version="1" encoding="utf-8" ?>
<html>
  <body>this string contains an e-acute: Ã©</body>
</html>

